style.go

```diff
@@ -183,7 +183,8 @@ func DefaultFill() *Fill {
 }
 
 func DefaultBorder() *Border {
-	return NewBorder("none", "none", "none", "none")
+	//return NewBorder("none", "none", "none", "none")
+	return NewBorder("", "", "", "")
```

Comment:
Please remove the commented code, also, what's the purpose of not using `"none"` anymore?